util/populate_dev_data.go

```diff
@@ -318,3 +321,152 @@ func FetchPendingRuns(wptdHost string) ([]shared.PendingTestRun, error) {
 	err := shared.FetchJSON(url, &pendingRuns)
 	return pendingRuns, err
 }
+
+func addFakeHistoryData(store shared.Datastore) {
+	// browser_name,browser_version,date,test_name,subtest_name,status
```

Comment:
This looks fine as a temporary change. Can we add a TODO comment here to import real data here when staging is populated with real data?